src/main/java/com/google/api/codegen/transformer/InitCodeTransformer.java

```diff
@@ -336,23 +352,15 @@ private InitCodeView buildInitCodeView(
     List<InitCodeLineView> argDefaultParams = new ArrayList<>();
     List<InitCodeLineView> argDefaultLines = new ArrayList<>();
     for (InitCodeNode param : sampleFuncParams) {
-      if (param.getLineType() == InitCodeLineType.ReadFileInitLine) {
-        InitCodeNode child = param.getChildren().get(InitCodeNode.FILE_NAME_KEY);
-        InitCodeLineView view = generateSimpleInitCodeLine(context, child, false);
-        argDefaultLines.add(view);
-        argDefaultParams.add(view);
-        orderedItems.remove(child);
-      } else {
-        List<InitCodeNode> paramInits = param.listInInitializationOrder();
-        argDefaultLines.addAll(generateSurfaceInitCodeLines(context, paramInits));
+      List<InitCodeNode> paramInits = param.listInInitializationOrder();
```

Comment:
Where did the functionality for `InitCodeLineTyoe.ReadFileInitLine` move to?

Reply:
[sampleFuncParams](https://github.com/googleapis/gapic-generator/pull/2480/files/9a446decd21b79e18260cebe2c04949663ac982a#diff-e3e4fdf548cccc11815a3f68b7849cf3R271) now handles that logic.